### PR TITLE
feat(schematics): add SCSS/CSS support via Prettier

### LINKED ISSUE #457: Feature request: format scss with prettier
It is great that nx uses prettier to format ts. However, prettier also supports formatting other file types, and I think it would be valuable to use prettier to format css/scss (and html also, once that is supported by prettier).

As a reference, I am currently doing this on my project's package.json:

```diff
-    "format:check": "./node_modules/.bin/nx format check",
-    "format:write": "./node_modules/.bin/nx format write",
+    "format-scss:check": "prettier '{apps,libs}/**/*.scss' --debug-check",
+    "format-scss:write": "prettier '{apps,libs}/**/*.scss' --write"`
+    "format-ts:check": "./node_modules/.bin/nx format check",
+    "format-ts:write": "./node_modules/.bin/nx format write",
+    "format:check": "npm run format-ts:check && npm run format-scss:check",
+    "format:write": "npm run format-ts:write && npm run format-scss:write",
```

---

On another topic, can someone please tell me why nx scripts use the relative path `./node_modules/.bin/nx` instead of just using `nx`? If I understand it correctly, running `npm run <script-name>` will look for executables in `node_modules/.bin/`, so why is it necessary to use a relative path?

### LINKED ISSUE #459: Feature request: css/scss linting
It would be great to add css/scss linting to the lint script in addition to linting typescript. This can easily be accomplished with [sass-lint](https://github.com/sasstools/sass-lint).

Install `sass-lint`
`npm install sass-lint --save-dev`

Edit the lint script in `package.json`:

```diff
-    "lint": "ng lint --format=stylish",
+    "lint-scss": "sass-lint -v -q",
+    "lint-ts": "ng lint --format=stylish",
+    "lint": "npm run lint-ts && npm run lint-scss",
```

Finally, create a `.sass-lint.yml` file in the root of the project, see sample:
https://github.com/sasstools/sass-lint/blob/6a71e29920a386fd97b986eb54ed0123890c1913/docs/sass-lint.yml

Also, see available rules: 
https://github.com/sasstools/sass-lint/tree/6a71e29920a386fd97b986eb54ed0123890c1913/docs/rules

### PR DESCRIPTION
**Pull request by hmagrini**
closes https://github.com/nrwl/nx/issues/459
closes https://github.com/nrwl/nx/issues/457

Love the project! Has helped out maintain my codebase neatly organized!

Hope you don't mind me adding this PR, I was planning on adding a workaround via npm scripts to my codebase directly, but figured it would take me the same time as opening this PR :)

Happy to add an option to opt-out (or opt-in) to the CLI. I'm unsure how opinionated Nx would like to be regarding non-TS files. Please let me know and I can include that on the PR 👍

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@nrwl/nx-source",
-  "version": "6.2.0",
+  "version": "6.2.1-beta.1",
```
Comment (vsavkin) [inline package.json]: Could you remove the first commit from this PR?

**Review by vsavkin — APPROVED** [review]
Looks good.

**Comment by vsavkin** [comment]
Thank you so much for this PR!

**Comment by vsavkin** [comment]
Great. Thank you

**Comment by katyapavlenko** [comment]
Hi! I can't fully understand how can I use this feature if I want to have separate prettier configs for scss and ts
Can you please point me to documentation?

**Pull request merged**